include/votca/tools/eigen.h

```diff
@@ -45,6 +49,7 @@
 #endif
 #include <Eigen/Eigen>
 #include <unsupported/Eigen/CXX11/Tensor>
+#include <unsupported/Eigen/FFT>
```

Comment:
Is there a reason to always include this? Why not just the correlate class?

Reply:
I thought it is maybe a good idea to make it available to all of votca, as people might want to use this functionality.

Reply:
because FFT could come up quite quickly in XTP as well

Reply:
but putting it into correlate would probably be enough

Reply:
I think FFT could be useful for further development and it is pretty well namespaced so I actually would make it available in all of votca

Reply:
what do you think?

Reply:
No preference just wondering.